consensus/cw_rating_calculator/cw_rating_dfs_impl.c

```diff
@@ -160,11 +159,13 @@ static retcode_t cw_rating_dfs_do_dfs_from_db(
           return res;
         }
       }
+      continue;
     }
+    hash_stack_pop(&stack);
```

Comment:
You have two `hash_stack_pop(&stack);` ?

Reply:
but only one will be executed since there's continue

Reply:
Alright, been confused by the brackets